v1.2 made edits for 1.2 release (#130)
mulesoft/docs-service-mesh: modules/ROOT/pages/upgrade-service-mesh.adoc

= Upgrade to Anypoint Service Mesh 1.2
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: service mesh, microservices, upgrading

> Anypoint Service Mesh v1.2 includes new features and architecture enhancements, such as the authorization mechanism aspect of the mixer is now implemented by leveraging Anypoint Service Mesh Envoy filters. Note that the architectural changes in the upgrade require some downtime.

Reviewer: ```suggestion
Anypoint Service Mesh v1.2 includes new features and architecture enhancements, such as leveraging Anypoint Service Mesh Envoy filters to implement the mixer authorization mechanism. Note that implementing these changes requires some downtime.
```
```suggestion
Anypoint Service Mesh v1.2 includes new features and architecture enhancements, such as the authorization mechanism aspect of the mixer is now implemented by leveraging Anypoint Service Mesh Envoy filters. Note that the architectural changes in the upgrade require some downtime.
```

Author: Done! Thanks!

Before you upgrade, ensure that your environment uses:

* Anypoint Service Mesh v1.1.x
* The most recent version of the Anypoint Service Mesh command line interface (CLI)
+
The CLI version must be v1.2.x and the Anypoint Service Mesh version must be v1.1.x.

> * Istio 1.7.x or 1.8.x or later for both control plane and data plane

Reviewer: Replace with
* Anypoint Service Mesh v1.1.x
* CLI v1.2.x 
* Istio 1.7.x or 1.8.x or later for both control plane and data plane
...but how can the CLI version be 1.2.x already, before the upgrade?

Reviewer 2: The CLI is standalone of the product version, to be able to upgrade to Service Mesh 1.2.x you need to use a CLI 1.2.x in order to do that

Author: Thanks for the clarification @dosten.

+
[NOTE] If you are using the EU control plane, you must specify the correct `--platformUri` flag for the upgrade.

> //Rodri to check whether we need the clean up post-upgrade steps

Reviewer 3: Remove this comment 😄

Author: Done! Thanks!
